samples/MvcSandbox/.gitignore

```diff
@@ -1,2 +0,0 @@
-# Ignore everything
```

Comment:
Basically `git` should ignore new files. But it will normally pick up any local changes made to files already in the index.